src/styles/blocks/_stars.scss

```diff
@@ -0,0 +1,37 @@
+.stars {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: space-between;
+
+  align-items: center;
+
+  &__container {
+    display: flex;
+
+    &-star {
+      background-image: url("../images/star.svg");
+      background-position: center;
+      background-repeat: no-repeat;
+      width: 16px;
+      height: 16px;
+    }
+
+    &-star:not(:first-child) {
+      margin-left: 4px;
+    }
+
+    &--4 &-star:nth-child(-n + 4) {
+      background-image: url("../images/star-active.svg");
+    }
```

Comment:
Add styles for other modifiers too.
![image](https://user-images.githubusercontent.com/96409563/222245236-08b391ae-05dd-4766-af17-dbe547b5d095.png)